### PR TITLE
build(dependabot): introduce

### PR DESCRIPTION
**Pull request by AugustinMauroy**
I propose to add dependabot to keep project healthy

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates
https://docs.github.com/ru/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates

### CODE REVIEW

**Comment by Pisyukaev** [comment]
It is very ineteresting, thanks you, I will thinking about it!

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 2
+enable-beta-ecosystems: true
+updates:
+    - package-ecosystem: npm
```
Comment (Pisyukaev) [inline .github/dependabot.yml:4]: In project used the ```pnpm```

Comment (AugustinMauroy) [reply .github/dependabot.yml:4]: Capture d’écran 2023-06-23 à 10 12 47

**Comment by Pisyukaev** [comment]
Thank you for your PR!
Was interesting read about dependabot update versions, but I like to control all the dependencies myself and update them when needed.

**Pull request closed without merging**